src/google/protobuf/compiler/cpp/cpp_helpers.h

```diff
@@ -205,9 +205,6 @@ inline const Descriptor* FieldScope(const FieldDescriptor* field) {
 std::string FieldMessageTypeName(const FieldDescriptor* field,
                                  const Options& options);
 
-// Strips ".proto" or ".protodevel" from the end of a filename.
-PROTOC_EXPORT std::string StripProto(const std::string& filename);
```

Comment:
I think we still need to keep this one around unfortunately, because within Google we have a bunch of code that calls this particular one. Could you perhaps make this one just delegate to the new version from code_generator.h?

Reply:
I have added a proxy method which is easy to remove once internal codebase will be adjusted.
Will this work?

Did not bother about ABI compatibility (i. e. PROTOC_EXPORT / placement the method into .cc file) as entire patch will break ABI anyway.